src/routes/api/data-cleanup/cleanup-reposyncstates.ts

```diff
@@ -24,7 +24,7 @@ export const RepoSyncStateCleanUpOrphanDataPost = async (req: Request, res: Resp
 		const cleanUpFromWhereSql = getCleanupSql(repoSyncStateId);
 		const sql = commitToDB === true ? getCommitInDBSql(cleanUpFromWhereSql) : getDryRunSql(cleanUpFromWhereSql);
 		const result = await RepoSyncState.sequelize?.query(sql);
-		const parsedResult = safeParseResult(result, commitToDB);
+		const parsedResult = result && safeParseResult(result, commitToDB);
```

Comment:
should we rather throw an exception explicitly if null/undefined/empty/whatever we are not expecting?